src/OpenAL/OpenToolkit.OpenAL/Extensions/EXT.Capture/Capture.cs

```diff
@@ -171,11 +170,21 @@ public static void CaptureSamples<TBuffer>(ALCaptureDevice device, TBuffer[] buf
         public static void CaptureSamples<TBuffer>(ALCaptureDevice device, Span<TBuffer> buffer, int sampleCount)
             where TBuffer : unmanaged
         {
+#if NETCOREAPP3_1
```

Comment:
Just a question, can we replace `NETCOREAPP3_1` for `NETCOREAPP`

Reply:
This file is deleted in the latest :)